### PR TITLE
Fix to bring PDF download link back for citations.

### PR DESCRIPTION
**Pull request by nchiasson-dgi**

### CODE REVIEW

File: uwsdora.module [hunk 1]
```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
     )
   );
   $output['citation.tab']['google_scholar_search']['#markup'] = $link;
+
+  // Adding PDF Download Link
+  $default_viewer = variable_get('islandora_scholar_viewers', array('default' => 'none',));
+  $default_viewer = $default_viewer['default'];
+  if (isset($object['PDF']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PDF']) && ($default_viewer == 'none') && !isset($output['citation.tab']['pdf_download']['#markup'])) {
```
Comment (IAMlKeno) [inline uwsdora.module]: * I don't think wrapping $default_viewer == 'none' in parentheses is necessary.
* For readability sake, it make help to break the if condition into separate lines 
```php
if (isset($object['PDF']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PDF']) && 
  ($default_viewer == 'none') && 
  !isset($output['citation.tab']['pdf_download']['#markup'])
) {
```

File: uwsdora.module [hunk 2]
```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
...
+    $filename = str_replace(":", "_", $object->id);
```
Comment (IAMlKeno) [inline uwsdora.module]: filename isn't used

File: uwsdora.module [hunk 3]
```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
...
+    $output['citation.tab']['pdf_download'] = array(
+      '#type' => 'item',
+      '#markup' => l(t('Download PDF'), "islandora/object/$object->id/datastream/PDF/download/citation.pdf", array('attributes' => array('class' => 'scholar-pdf-download    -link'))),
```
Comment (IAMlKeno) [inline uwsdora.module]: * Extra space in class name "scholar-pdf-download    -link"
* For readability sake, could probably break the l function call to multiple lines - like:
```php
'#markup' => l(
    t('Download PDF'), 
    "islandora/object/$object->id/datastream/PDF/download/citation.pdf", 
    array(
      'attributes' => array('class' => 'scholar-pdf-download-link'),
    )
  ),
```

File: uwsdora.module [hunk 4]
```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
     )
   );
   $output['citation.tab']['google_scholar_search']['#markup'] = $link;
+
+  // Adding PDF Download Link
```
Comment (IAMlKeno) [inline uwsdora.module]: may want to run drush-no-test-empty uwsdora to catch missed coding standards

**Review by IAMlKeno — CHANGES_REQUESTED** [review]

File: uwsdora.module (hunk 1 above)
Comment (nchiasson-dgi) [reply uwsdora.module]: Implemented readability feedback

File: uwsdora.module (hunk 2 above)
Comment (nchiasson-dgi) [reply uwsdora.module]: Removed filename variable.

File: uwsdora.module (hunk 4 above)
Comment (nchiasson-dgi) [reply uwsdora.module]: Ran and fixed all coding standards issues.

**Pull request merged**